src/test/resources/application.yaml

```diff
@@ -7,6 +7,9 @@ drools:
 management:
   security:
     enabled: ${MANAGEMENT_SECURITY_ENABLED:true}
+    rules:
+      jwtSecret: "FakeTestSecret"
```

Comment:
Make sure that the secrets you use will work with real services. It is important for integration tests which will fail to authenticate if secrets are incorrect